rockcraft.yaml

```diff
@@ -20,7 +20,7 @@ parts:
   certificates:
```

Comment:
question: What is the best practice? Should it be in a separate part or should it be in the `hydra` part?

Reply:
The recommendation is to keep it separate for clarity. All parts are merged on build